scripts/build_shapes.py

```diff
@@ -812,7 +863,12 @@ def gadm(
 
     country_shapes = countries(countries_list, geo_crs, update, out_logging)
 
-    country_shapes.reset_index().to_file(snakemake.output.country_shapes)
+    # there may be "holes" in the countries geometry which cause troubles along the workflow
+    # e.g. that is the case for enclaves like Dahagram–Angarpota for IN/BD
+    country_shapes_valid = country_shapes.apply(
```

Comment:
I'd overwrite the country_shapes variable or pipe the function to clean the code

Reply:
Done!